data/data/vsphere/main.tf

```diff
@@ -79,7 +81,7 @@ module "bootstrap" {
   ignition      = var.ignition_bootstrap
   resource_pool = data.vsphere_compute_cluster.cluster.resource_pool_id
   datastore     = data.vsphere_datastore.datastore.id
-  folder        = vsphere_folder.folder.path
+  folder        = var.vsphere_folder
```

Comment:
@patrickdillon in the scenario that we create a folder this isn't an issue?
Since there is no dependency on the folder resource to happen before virtual machine resource.

Reply:
There's no error (this is the scenario tested by CI), but that seems merely by chance. I have fixed this by adding a local variable. Watching CI test now...